optimum/commands/export/openvino.py

```diff
@@ -364,6 +371,14 @@ def run(self):
             if not is_nncf_available():
                 raise ImportError("Applying quantization requires nncf, please install it with `pip install nncf`")
 
+            if (self.args.weight_format == "cb4" or self.args.quant_mode == "cb4_f8e4m3") and is_nncf_version(
+                "<=", "2.17"
+            ):
+                raise ImportError(
+                    "Codebook quantization is currently supported only with NNCF develop. "
+                    "Please run `pip install git+https://github.com/openvinotoolkit/nncf.git`."
+                )
+
             default_quantization_config = get_default_quantization_config(
```

Comment:
do we need to check here as an error will be raised  in all cases when creating a quantization configuration with `get_default_quantization_config` no ?